src/util/SslPatch.py

```diff
@@ -6,6 +6,7 @@
 import sys
 import ctypes
 import ctypes.util
+import ssl
```

Comment:
@shortcutme is this the right way to resolve the undefined name errors at https://travis-ci.org/cclauss/itinerant-tester/jobs/259296035#L314-L322